config/config.go

```diff
@@ -86,9 +86,9 @@ type FlagConfig struct {
 
 // Load loads a config toml
 func (cfg *Config) Load(file string) error {
-	_, err := os.Stat(file)
+	_, err := os.Stat(Expand(file))
```

Comment:
It's also not necessary to call Expand here and in the readfile below I think.

Look at the code, config.Load is only called in one place, and there it is set to the default location, unless someone passes a command line argument.

I think it is fair to assume that the command line argument will never be a relative path.

Reply:
It's better to be explicit and consistent across all instances, I think. Otherwise, we'd be circling back to "why not expand here" down the road .

Plus expand isn't computationally expensive in any way

Reply:
Totally agree with being explicit and consistent. I'm just against calling it **unnecessarily**, which is a different thing than if it were **optional**.

If we're expanding where we shouldn't expand, it's bad logic. Confusing to the reader, potentially cause for bugs in the future, etc.

Let me think about this some more. There must be a better way. <insert Raymond Hettinger meme>

Reply:
I'm gonna do a few modifications to this, pushing in bit. A nice compromise I found is just using a variable here - makes it clear that the path to be used is the absolute. So anyone modifying this in the future will reuse absPath basically, instead of having to remember to call Expand. This reduces the risk enough to forget about this issue in my eyes 🙂.

Sorry hard to communicate these things exactly - it's more an art trying to balance between preventing future developers from making preventable mistakes, readability, complexity and performance.